### ISSUE TITLE
Preserve threads on restart

### ISSUE THREAD

**Issue by virus-found** [wee-slack/wee-slack#743]
Hi, thanks for this plugin, I use it every day.

When I restart weechat I lose all my manually opened threads.
Is there a way to preserve them?

**Comment by trygveaa** [comment]
No, at the moment there is unfortunately not any way to preserve them.

I'll keep this issue open as a feature request.

**Comment by krzkrzkrz** [comment]
Would love to have this feature as well

**Comment by pallix** [comment]
It would be great to have that
